cmd/nri-kube-events/main.go

```diff
@@ -27,7 +27,7 @@ import (
 
 const (
 	integrationName    = "nri-kube-events"
-	integrationVersion = "1.0.0"
```

Comment:
can we make this automatic with build flags? https://github.com/newrelic/nri-kube-events/issues/26

Reply:
the is also eventRouterVersion for some reason that is set statically

Reply:
I bumped those numbers as a part of the version bump, but I'm afraid that making them dynamic is not as easy as copy-pasting from one of our other repos since we don't use goreleaser for this one. I'd rather descope that and leave it for another PR.

Reply:
:fine for me!